games-emulation/4do_libretro/4do_libretro-1.3.2.4.20180219git.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="A port of 4DO, a 3DO emulator to the libretro architecture"
+DESCRIPTION="4DO is an open-source, low-level emulator for the Panasonic 3DO, based on the FreeDO source code."
+HOMEPAGE="http://www.fourdo.com/"
+COPYRIGHT="2011-2018 the 4DO/FreeDO team, the libretro team"
+LICENSE="
+	FreeDO
+	GNU LGPL v3
+	"
```

Comment:
```diff
-LICENSE="
-	FreeDO
+LICENSE="FreeDO
-	GNU LGPL v3
-	"
+	GNU LGPL v3"
```
(I'm not 100% sure. I'll do some metrics to see if we really have more recipes with the compact style I'm suggesting.)